### ISSUE TITLE
Make id an Int64 using bigserial

### ISSUE THREAD

**Issue by paulcsmith** [luckyframework/migrator#20]
See #14 

Will require changing LuckyRecord to use Int64 for the id. Right now it uses Int32

**Comment by mikeeus** [comment]
I just realized that I used an `Int64` in the `belongs_to` method which breaks associations that use `Int32` for ids.

https://github.com/luckyframework/lucky_migrator/blob/133d3320178e435bb942c5f7b34b3a2183840bc3/src/lucky_migrator/create_table_statement.cr#L111

I have no clue what I was thinking when doing that. Either I need to return this to `Int32` or work on changing `LuckyRecord` to using `Int64`. What do you think I should do?

**Comment by paulcsmith** [comment]
Good catch! I think for now it would be best to change it to Int32 so it can be released without having to make changes to migratory and LuckyRecord
